dev/conformance/conformance-tests/set-st-alone-mergeall.json

```diff
@@ -13,15 +13,16 @@
           "database": "projects/projectID/databases/(default)",
           "writes": [
             {
-              "transform": {
-                "document": "projects/projectID/databases/(default)/documents/C/d",
-                "fieldTransforms": [
-                  {
-                    "fieldPath": "a",
-                    "setToServerValue": "REQUEST_TIME"
-                  }
-                ]
-              }
+              "update": {
+                "fields": {},
+                "name": "projects/projectID/databases/(default)/documents/C/d"
+              },
+              "updateTransforms": [
```

Comment:
How much work would it be to preserve the initial Proto for the case where we only have transforms and no updates? That produces a cleaner and more minimalist Proto.